static/schemas/2.0/Node.2.0.json

```diff
@@ -0,0 +1,35 @@
+{
+    "title": "Node.2.0",
+    "definitions": {
+        "Node": {
+            "description": "A node discovered by RackHD",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the node",
+                    "type": "string"
+                },
+                "type": {
+                    "description": "Type of node",
+                    "type": "string",
+                    "enum": ["compute", "switch", "dae", "pdu", "mgmt", "enclosure"]
+                },
+                "obmSettings": {
```

Comment:
This is a first pass and the obmSettings/snmpSettings/bootSettings will have their own detailed schemas later?